conf/messages.en

```diff
@@ -98,7 +98,7 @@ accountingPeriod.error.required = The accountingPeriod must include {0}
 accountingPeriod.error.invalid = Enter a real AccountingPeriod
 accountingPeriod.change.hidden = AccountingPeriod
 accountingPeriod.forExample=For example, 27 3 2021
-accountingPeriod.accountingPeriodCannotBeLongerThan12M=Your accounting period cannot be longer than a full year and is normally the same as the financial year covered by your company's annual accounts.
+accountingPeriod.accountingPeriodCannotBeLongerThan12M=Your accounting period cannot be longer than 12 months and is normally the same as the financial year covered by your company''s annual accounts.
```

Comment:
Why is this being changed? Design wants this to be a full year instead of 12 months